src/utils/validations/pwValidator.ts

```diff
@@ -0,0 +1,25 @@
+function isNot4Digit(num: number) {
+  return num.toString().length !== 4;
+}
+
+interface Validator {
+  test: (num: number) => boolean;
+  message: string;
+}
+
+const validators: Validator[] = [
+  {
+    test: isNot4Digit,
+    message: '비밀번호 4자리를 입력해주세요.',
+  },
+];
```

Comment:
이 부분도 위랑 조금 겹치는 것 같아서 뭔가 pw, name을 키로 구분해서 파일을 하나로 합쳐도 좋을 것 같아요 ! 
```suggestion
function hasBlank(str: string) {
  return str.trim() !== str;
}

function isEmpty(str: string) {
  return str === '';
}

function isNot4Digit(num: number) {
  return num.toString().length !== 4;
}

const validators: Validator[] = {
"pw": [
  {
    test: isNot4Digit,
    message: '비밀번호 4자리를 입력해주세요.',
  }
],
"name":  [{
    test: hasBlank,
    message: '이름 앞 뒤, 공백을 제외하고 입력해주세요.',
  },
  {
    test: isEmpty,
    message: '이름을 입력해주세요.',
  }]
}

export function validator(type: "pw" | "name", value) {
  return validators[type].every(({ test, message }) => {
    if (test(pw)) {
      throw new Error(message);
    }
    return true;
  });
}
```

Reply:
굿잡 반영했어욤

as랑 any 난사..
[4d361b8](https://github.com/TeamYouniverse/Youniverse-Web-Release/pull/11/commits/4d361b86d4f6154668a36e07fac95125ff0eceef)